### PR TITLE
Adds a fluff item for Ssscratches

### PR DESCRIPTION
**Pull request by FlattestGuitar**
```
/obj/item/clothing/suit/toggle/lasshy_wings
Nice wings that flap. So magical.

ckey: Ssscratches
name: Lasshy
```

### CODE REVIEW

**Comment by Crazylemon64** [comment]
now to add a vending machine to pick up wings from /s

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -830,3 +830,31 @@
...
+	icon_state = "lasshy_wings"
+	item_state = "lasshy_wings"
+	actions_types = list(/datum/action/item_action/toggle_wings)
+	var/icon/mob
+
+/obj/item/clothing/suit/toggle/lasshy_wings/update_icon(var/mob/living/carbon/human/user) //holy pickles, copied over from /obj/item/clothing/head/kitty
+	if(!istype(user))
+		return
+	var/obj/item/organ/external/head/head_organ = user.get_organ("head")
+	mob = icon("icon" = 'icons/mob/suit.dmi', "icon_state" = icon_state)
```
Comment (tigercat2000) [inline code/modules/customitems/item_defines.dm]: This variable isn't really accurately named, and it's a predefined path, so the syntax highlighting is fucked up. I would recommend changing it to `blended_icon`. It also doesn't really need to be defined as a static part of the path, why not just declare and initialize a new icon in this proc? It's not being used anywhere in the future.

**Comment by Necaladun** [comment]
i dunno what tiger said there so change it so tiger doesnt yell at you pls

As for the item itself - what is this supposed to be? 
I get that it's a pair of butterfly wings...but wat?

**Comment by KasparoVy** [comment]
IDK either. :grin: :ok_hand:

**Comment by taukausanake** [comment]
It looks really weird. Not bad but that's a lot of icon for the tile. Also no directional shots?

**Comment by SamHPurp** [comment]
This is the sort of shit get servers get mocked for.

**Comment by Fruerlund** [comment]
I have to agree with others. This is just. 


Eh.

**Comment by Spacemanspark** [comment]
I think this is a bit much...

**Comment by Anticept** [comment]
This would make a great meme item.

But if it's something they intend to wear all the time...

**Comment by KasparoVy** [comment]
Surely sanity will prevail

**Comment by Fox-McCloud** [comment]
@TullyBurnalot @Necaladun @FreeStylaLT

**Comment by Rumiluntti** [comment]
Jesus christ how absolutely horrifying

**Comment by FlattestGuitar** [comment]
As per the Heads' decision on this one.

**Pull request closed without merging**